dev/documentation/en/datamodel-settings.md

```diff
@@ -0,0 +1,80 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Settings - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: "Settings"
+
+Settings are additional property values associated with an instance of an entity type.
+
+## The Purpose of Settings
+
+Settings are an important concept in the PKP data model.
+They support two of the [overall goals](datamodel):
+
+### Localization
+
+If some logical attribute (say, a User's `familyname`) is to be localizable,
+the attribute will be stored in settings.
+
+For example a User from Japan might have one `familyname` in Kanji for the `ja_JP` locale
+and another in Latin characters for the `en_US` locale (and the rest of the world).
+For the same one User object, two settings objects will be created to represent
+the two localized versions of the User `familyname` property.
+
+OJS (etc.) knows about these settings and will create and use them regularly.
+See [Entities](architecture-entities.md) for what localization looks like
+at the PHP level.
+
+
+### Extensibility
+
+In order to provide its functionality, a plugin may need to add some information
+to entities of existing types.
+
+For example a simple monitoring service plugin may want to add a `processed` flag
+to all Submission objects.
+
+OJS (etc.) does not know about these settings and will not touch them.
+The plugin has full control over them if it makes sure not to use a setting name
+also used somewhere else.
+
+## How Settings Work
```

Comment:
This section duplicates the [database](https://docs.pkp.sfu.ca/dev/documentation/en/architecture-database) section and can be removed.